spec/formatting.md

```diff
@@ -663,17 +663,44 @@ These are divided into the following categories:
     > Example invalid messages resulting in a Missing Fallback Variant error:
     >
     > ```
-    > match {$one}
+    > match {$one :func}
     > when 1 {Value is one}
     > when 2 {Value is two}
     > ```
     >
     > ```
-    > match {$one} {$two}
+    > match {$one :func} {$two :func}
     > when 1 * {First is one}
     > when * 1 {Second is one}
     > ```
 
+  - **Missing Selector Annotation errors** occur when the message
```

Comment:
Don't dig the formatting hole deeper. When you introduce the term, do so in the form:

```suggestion
A **_Missing Selection Annotation error_** is an error that occurs when the _message_
```

That is, the form is:

> A **_thing_** is {definition} with _terms_ italicized.